include/ppx/tri_mesh.h

```diff
@@ -162,6 +162,7 @@ class TriMesh
     const float3& GetBoundingBoxMin() const { return mBoundingBoxMin; }
     const float3& GetBoundingBoxMax() const { return mBoundingBoxMax; }
 
+    void     PreallocateForTriangleCount(size_t triangleCount, bool enableColors, bool enableNormals, bool enableTexCoords, bool enableTangents);
```

Comment:
comment maybe? Bigwheels hasn't been great on comments, but we are trying to change that, at least in headers.

Reply:
Added comment.